### ISSUE TITLE
Automatische Tests und Reproducible Builds

### ISSUE THREAD

**Issue by Nos-** [fsfw-dresden/usb-live-linux#60]
Wäre praktisch. Vielleicht per Travis CI per Nightly-Build

**Comment by cknoll** [comment]
@Nos- hast Du Lust das mal auszuprobieren vielleicht erstmal mit einem separaten repo?

Relevant wären folgende Sachen:
- kann man auf Travis virtuelle Maschinen ausführen?
- wie stößt man am besten in der VM die Tests an (Autostart? SSH?)
- wie kann man komplexe Nutzereingaben testen (Klick auf Latex-Vorlage -> Klick auf öffnen mit texstudio -> klick auf kompilieren -> Abfragen, ob Fehlermeldungen gekommen sind?)
- kriegt man ggf. das gebaute Image irgendwohin geladen (ohne dass man einen privaten SSH-Schlüssel in ein öffentliches Repo legen muss? -> dann könnte man Travis als Build-Dienst benutzen)

**Comment by Nos-** [comment]
Ja, gern. Allerdings wird es erst in den nächsten Tagen werden,
